{"uuid":"50c6521d-f230-42dc-9a09-d64c0d4f0ec0","historyId":"b3cc92e95594db797b888bcfad924610","fullName":"testcases.AllTestTogether.test_POSTaddObject","labels":[{"name":"package","value":"testcases.AllTestTogether"},{"name":"testClass","value":"testcases.AllTestTogether"},{"name":"testMethod","value":"test_POSTaddObject"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"API_Live_Automation"},{"name":"subSuite","value":"testcases.AllTestTogether"},{"name":"host","value":"shubraj-22024452H"},{"name":"thread","value":"21932@shubraj-22024452H.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test_POSTaddObject","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Row.iterator()\" because \"row\" is null","trace":"java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Row.iterator()\" because \"row\" is null\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:809)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:734)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Row.iterator()\" because \"row\" is null\r\n\tat utility.ExcelDataProvider.getRowData(ExcelDataProvider.java:19)\r\n\tat utility.DataProviderClass.excelDataProvider(DataProviderClass.java:17)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1722798244217,"stop":1722798244221}